control/control.go

```diff
@@ -90,8 +93,34 @@ type pluginControl struct {
 	grpcServer  *grpc.Server
 	closingChan chan bool
 	wg          sync.WaitGroup
+
+	// taskView mutex
+	view *sync.Mutex
```

Comment:
Perhaps it would be more readable if you used `[variable being guarded]Mutex` as a name.